exercises/concept/savings-account/savings_account_test.rb

```diff
@@ -59,33 +59,33 @@ def test_annual_balance_update_for_empty_start_balance
   end
 
   def test_annual_balance_update_for_small_positive_start_balance
-    assert_in_delta 0.000_001_005, SavingsAccount.annual_balance_update(0.000_001), 0.000_000_0001
+    assert_in_delta 0.000_001_005, SavingsAccount.annual_balance_update(0.000_001), 0.000_000_000_1
   end
 
   def test_annual_balance_update_for_average_positive_start_balance
     assert_in_delta 1_016.21, SavingsAccount.annual_balance_update(1_000.0), 0.01
   end
 
   def test_annual_balance_update_for_large_positive_start_balance
-    assert_in_delta 1_016.2_101_016_209_999, SavingsAccount.annual_balance_update(1_000.0001), 0.0_000_000_000_01
+    assert_in_delta 1_016.2_101_016_209_999, SavingsAccount.annual_balance_update(1_000.000_1), 0.000_000_000_001
   end
 
   def test_annual_balance_update_for_huge_positive_start_balance
     assert_in_delta 920_352_587.26_744_292_868_451_875,
       SavingsAccount.annual_balance_update(898_124_017.826_243_404_425),
-      0.00_000_000_000_000_001
+      0.000_000_000_000_000_01
```

Comment:
Can you move this up one line, as is consistent with the others as well, and line 42 could use the underline treatment as well.

Reply:
Sure.